Modules/Core/Common/include/itkLaplacianOperator.h

```diff
@@ -97,8 +97,8 @@ class ITK_TEMPLATE_EXPORT LaplacianOperator : public NeighborhoodOperator<TPixel
     Superclass::PrintSelf(os, i.GetNextIndent());
   }
 
-  /** Sets the weights that are applied to the derivative in each axial
-   *  direction when the kernel is computed.  These weights are all 1.0 by
+  /** Set/Get the weights that are applied to the derivative in each axial
+   *  direction when the kernel is computed. These weights are all 1.0 by
    *  default. This method must be called BEFORE CreateOperator */
   void
   SetDerivativeScalings(const double * s);
```

Comment:
Still not convinced by this and its getter method due to this:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/test/itkNeighborhoodOperatorTest.cxx#L150
If somebody happens to have a proposal, I'd consider it.